backend/config.js

```diff
@@ -11,6 +11,7 @@ const configData = envSchema({
     .prop("AUTH_DOMAIN", S.string().required())
     .prop("AUTH_SECRET_KEY", S.string().required())
     .prop("AUTH_STATE", S.string().required())
+    .prop("ENVIRONMENT", S.string().default("dev").required())
```

Comment:
I would avoid setting a default value for an environment. If it is not defined, that would seem like an error that we should know about and fix it.